From: Brian Murray <brian.murray@canonical.com>
Date: Tue, 1 Nov 2016 12:35:45 +0800
Subject: Include package version in package data

Apport includes the package and version in the package data for any report that
it creates. The Ubuntu Error Tracker expects the data to follow this format,
and should not have to look up the package version in different fields, so this
special hook should follow the format.
---
 dkms_apport.py | 12 ++++++------
 1 file changed, 6 insertions(+), 6 deletions(-)

diff --git a/dkms_apport.py b/dkms_apport.py
index c495967..fad0126 100755
--- a/dkms_apport.py
+++ b/dkms_apport.py
@@ -62,18 +62,18 @@ if options.kernel:
 make_log=os.path.join('/var','lib','dkms',options.module,options.version,'build','make.log')
 
 report = apport.Report('Package')
-report['Package'] = package
-try:
-    report['SourcePackage'] = apport.packaging.get_source(package)
-except ValueError:
-    sys.stderr.write('ERROR (dkms apport): unable to determine source package for %s\n' % package)
-    sys.exit(3)
 try:
     version = packaging.get_version(package)
 except ValueError:
     version = '(not installed)'
 if version is None:
     version = '(not installed)'
+report['Package'] = '%s %s' % (package, version)
+try:
+    report['SourcePackage'] = apport.packaging.get_source(package)
+except ValueError:
+    sys.stderr.write('ERROR (dkms apport): unable to determine source package for %s\n' % package)
+    sys.exit(3)
 
 if report['SourcePackage'] == 'fglrx-installer':
     fglrx_make_log = os.path.join('/var','lib','dkms',options.module,options.version,'build','make.sh.log')
